packages/fxa-settings/src/pages/InlineTotpSetup/index.stories.tsx

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import InlineTotpSetup from '.';
+import { Meta } from '@storybook/react';
+import { MozServices } from '../../lib/types';
+import AppLayout from '../../components/AppLayout';
+import { MOCK_CODE, MOCK_EMAIL } from './mocks';
```

Comment:
Not a blocker since we're doing this elsewhere and probably need to clean up other files as well but can we use the mock primary email on `Account` instead of this one?

Reply:
So, following the patterns in other existing views, this expects `email` to come in as a string, and typescript did not like passing `primaryEmail` in. It might be helpful to come to a final decision on where we're going to keep our mocks -- having the local mock file and also the shared mock file seems a little messy, but the shared mock also doesn't have everything we need yet